docs/Android.md

```diff
@@ -204,28 +194,14 @@ device. As in part one, you'll need to connect your Android device via USB:
 ```
 $ utils/build-script \
   -R \                                           # Build in ReleaseAssert mode.
-  -T \                                           # Run all tests.
+  -T --host-test \                               # Run all tests, including on the Android host.
   --android \                                    # Build for Android.
-  --android-ndk ~/android-ndk-r13 \              # Path to an Android NDK.
+  --android-ndk ~/android-ndk-r21 \              # Path to an Android NDK.
+  --android-arch armv7 \                         # Optionally specify Android architecture, alternately aarch64
   --android-ndk-version 21 \
   --android-icu-uc ~/libicu-android/armeabi-v7a/libicuuc.so \
   --android-icu-uc-include ~/libicu-android/armeabi-v7a/icu/source/common \
   --android-icu-i18n ~/libicu-android/armeabi-v7a/libicui18n.so \
-  --android-icu-i18n-include ~/libicu-android/armeabi-v7a/icu/source/i18n/
-```
-
-## Build Android Toolchain
-
-This toolchain will generate the .so and .swiftmodule files of the Swift standard library and Foundation framework for the Android environment, armv7 architecture. Those files are needed when building any Swift library to be included in an application for Android.
-
```

Comment:
This last section was just added with #12530 after two years in limbo, but the referenced build-toolchain script hasn't been updated since then either and most likely won't work, so just take this out.

Reply:
@amraboelela, let me know if it works.